packages/world/src/World.js

```diff
@@ -15,6 +16,10 @@
  */
 import React from 'react';
 
-export default function World() {
-  return <span>World</span>;
+type Props = {|
+  worldLabel?: string
```

Comment:
Suggest to rename `worldLabel` prop to `text`.  The word `Label` has a connotation to it.  The `world` part is redundant.